gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Is it necessary to set xmx here since we got this `org.gradle.jvmargs=-Xmx768m` in file gradle.properties? If we are going to use 768m as the xmx, we should just remove the 64m setting to avoid confusion.

Reply:
Those changes are generated automatically by Gradle when doing an upgrade